css/main.css

```diff
@@ -319,44 +222,21 @@ textarea {
     }
 }
 
-/*=============================================================
-Dev Challenge CSS - Version 2.0
-===============================================================
-*/
+/* ==========================================================================
+   Author's custom styles
+   ========================================================================== */
 
 body {
-    width:960px;
-    padding:0 20px 0 20px;
-
+    margin-top:25px;
 }
 
-
 header {
-    width:100%;
     margin:25px 0 10px 0;
-    display:block;
-    height:75px;
-
+    height:100px;
```

Comment:
If it's absolutely necessary to declare a height on a section with child elements, it's better practice to use 'min-height' so that at smaller screen widths the page flow isn't interrupted.